src/Castle.Services.Logging.NLogIntegration/NLogFactory.cs

```diff
@@ -32,14 +32,14 @@ public class NLogFactory : AbstractLoggerFactory
 		///   Initializes a new instance of the <see cref="NLogFactory" /> class.
 		/// </summary>
 		public NLogFactory()
-			: this(defaultConfigFileName)
+			: this(true)
```

Comment:
I've noticed that the `ExtendedNLogFactory` has very similar constructors as `NLogFactory`. Should the same change be applied there, too, for consistency?

Reply:
Good catch. Have now also updated `ExtendedNLogFactory`